sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightPlugin.kt

```diff
@@ -159,7 +159,6 @@ open class SqlDelightPlugin : Plugin<Project> {
               compilationUnit.extraOpts("-linker-options", "-lsqlite3")
               project.tasks.named(compilationUnit.compileAllTaskName).configure { it.dependsOn(task) }
               project.tasks.named(compilationUnit.compileKotlinTaskName).configure { it.dependsOn(task) }
-              project.tasks.named(compilationUnit.linkAllTaskName).configure { it.dependsOn(task) }
```

Comment:
```
kotlin {
  iosX64('ios') {
    binaries {
      framework()
    }
  }
}
```

If we create output binaries with new dsl as above, link task for that binary can be achieved via `compilationUnit.target.binaries.forEach { it.linkTask }`.
So I think we should care about that too.

Reply:
awesome - thanks